GitTfs.Setup/GitTfs.Setup.csproj

```diff
@@ -98,6 +98,13 @@
     <WCFMetadata Include="Service References\" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Choose>
```

Comment:
Please revert all the changes of this files. Blame paket but there is just noise here :(